### PR TITLE
fix spectral lint failure

### PR DESCRIPTION
**Pull request by hilary**
quick lint failure fix to unblock other work

### CODE REVIEW

File: dist/Lob-API-bundled.yml [hunk 1]
```diff
@@ -526,7 +526,7 @@ components:
           description: >
             Geographic direction preceding a street name (`N`, `S`, `E`, `W`,
             `NE`, `SW`, `SE`, `NW`).
-          example: ''
+          example: 'N'
```
Comment (sudoku-lord) [inline dist/Lob-API-bundled.yml:529]: I believe this example will show up in the example address given (the 185 BERRY ST one), which doesn't actually have any directionals IRL. Should we change the address to one which does have pre- and post-directionals, which would be a break from the current docs, or is it okay to put in a directional where there isn't one?

File: dist/Lob-API-bundled.yml [hunk 2]
```diff
@@ -555,7 +555,7 @@ components:
           description: >
             Geographic direction following a street name (`N`, `S`, `E`, `W`,
             `NE`, `SW`, `SE`, `NW`).
-          example: ''
+          example: 'N'
```
Comment (sudoku-lord) [inline dist/Lob-API-bundled.yml:558]: Same question as the one above.

File: resources/us_verifications/models/components.yml [hunk 3]
```diff
@@ -51,7 +51,7 @@ properties:
     description: >
       Geographic direction preceding a street name (`N`, `S`, `E`,
       `W`, `NE`, `SW`, `SE`, `NW`).
-    example: ""
+    example: "N"
```
Comment (sudoku-lord) [inline resources/us_verifications/models/components.yml:54]: Same question as above, and ditto for the next one.

File: dist/Lob-API-bundled.yml (hunk 1 above)
Comment (hilary) [reply dist/Lob-API-bundled.yml]: Ok, so I looked at the documentation again, and it appears to me as if what we need is to include the empty string in the enum list, as both pre and post directionals are required.

File: dist/Lob-API-bundled.yml (hunk 2 above)
Comment (hilary) [reply dist/Lob-API-bundled.yml]: Same answer as above :)

**Review by sudoku-lord — APPROVED** [review]

**Pull request merged**